authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -93,7 +93,7 @@ public class WebSecurityConfig {
           // swagger-ui endpoint
           "/swagger-ui/**",
           "/v3/api-docs/**",
-          "/camunda-api-docs/**",
+          "/rest-api.yaml",
```

Comment:
❓ Did we simply miss to adjust this with the previous PR when we moved to the Spring static resolver?

Reply:
This is where we moved to the static resource mechanism, we simply missed this there, right: https://github.com/camunda/camunda/pull/30129?

Reply:
I was fooled by the cache in my machine, I missed out this update and when I tested I havent relize the swagger-ui 

Yes, you are right :)